spec/rubocop/cop/factory_bot/create_list_spec.rb

```diff
@@ -71,12 +71,25 @@
       RUBY
     end
 
-    it 'ignores n.times when create call does have method calls' do
+    it 'ignores n.times when create call does have method calls ' \
+       'and repeat multiple times' do
       expect_no_offenses(<<~RUBY)
         3.times { |n| create :user, repositories_count: rand }
       RUBY
     end
 
+    it 'ignores n.times when create call does have method calls ' \
```

Comment:
Flags?

Reply:
Ah! Thank you very much. I updated this PR.